core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java

```diff
@@ -64,7 +64,7 @@ public ServiceRegistryLoader(Injector injector, ServiceRegistry registry) {
         }
         if (all.size() > 1) {
             throw new IllegalStateException(
-                "Multiple service implementations found for " + serviceClass + ". " + toClassString(all));
+                "Multiple service implementations found for " + serviceClass + ": " + toClassString(all));
```

Comment:
Would it make it more clear add some white spaces as well? `" : "`

Reply:
Not sure, that wouldn't be grammatically correct I think

Reply:
Ok :) it was just a comment :)

Reply:
Not sure if grammar has anything to do here :)